src/entity.unit.ts

```diff
@@ -163,6 +164,21 @@ describe("Entity", () => {
 			expect(person.Species).toBe("Homo sapiens");
 		});
 
+		it("cannot initialize reference properties to undefined", () => {
+			const person = Types.Person.meta.createSync({ Id: "1", Movie: undefined });
+			const movieProp = Types.Person.meta.getProperty("Movie");
+			expect(Property$pendingInit(person, movieProp)).toBe(true);
+			expect(person.Movie).toBe(null);
+			// expect(Property$pendingInit(person, movieProp)).toBe(false);
+		});
+
+		it("can initialize reference properties to null", () => {
+			const person = Types.Person.meta.createSync({ Id: "1", Movie: null });
+			const movieProp = Types.Person.meta.getProperty("Movie");
+			expect(Property$pendingInit(person, movieProp)).toBe(false);
```

Comment:
Test: Ensure that `pendingInit` is `false` after initializing the property to `null`